src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs

```diff
@@ -236,5 +239,49 @@ public async Task<IActionResult> UpdateDraftApprenticeship(long cohortId, long a
 
             return Ok();
         }
+
+        [HttpGet]
+        [Route("provider/{providerId}/unapproved/{cohortId}/apprentices/{draftApprenticeshipId}/edit/prior-learning-data")]
+        public async Task<IActionResult> GetPriorLearningData(long cohortId, long draftApprenticeshipId)
+        {
+            var result = await _mediator.Send(new GetEditDraftApprenticeshipPriorLearningDataQuery(cohortId,draftApprenticeshipId));
+
+            if (result == null)
+            {
+                return NotFound();
+            }
+
+            return Ok(result);
+        }
+
+        [HttpPost]
+        [Route("provider/{providerId}/unapproved/{cohortId}/apprentices/{draftApprenticeshipId}/edit/prior-learning-data")]
+        public async Task<IActionResult> PriorLearningData(long cohortId, long draftApprenticeshipId, [FromBody] InnerApi.Requests.AddPriorLearningDataRequest request)
```

Comment:
An inner api request should not be a parameter on an outer api endpoint.

Reply:
fixed